pages/speaking/cfp.vue

```diff
@@ -0,0 +1,216 @@
+<template>
+    <div class="px-64 py-24">
+        <h1 class="text-3xl my-8 text-center">{{ $t('title') }}</h1>
+        <i18n path="intro" tag="p" class="my-6"></i18n>
+
+        <!-- Important Dates -->
+        <i18n path="importantDates.0" tag="h2" class="text-2xl my-8"></i18n>
+        <i18n
+            v-for="(date, i) in $t('importantDates.1')"
+            :key="i"
+            :path="`importantDates.1.${i}`"
+            tag="p"
+            class="my-1"
+        >
+            <template #AoE>
+                <ext-link
+                    href="https://www.timeanddate.com/worldclock/converter.html?iso=20210427T115900&p1=tz_aoe&p2=241&p3=1440"
+                    :highlight="true"
```

Comment:
Remove unecessary `true`

```suggestion
                    highlight
```

Reply:
updated